app/models/user.rb

```diff
@@ -6,6 +6,42 @@ class User < ApplicationRecord
          :confirmable
 
   has_many :accounts, dependent: :destroy
+
+  def self.export_for_firebase
```

Comment:
Assignment Branch Condition size for export_for_firebase is too high. [35.36/25]<br>Method has too many lines. [33/15]